exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/writer/AbstractTupleWriter.java

```diff
@@ -143,6 +144,8 @@ public void dump(HierarchicalFormatter format) {
     ObjectWriter addColumn(TupleWriter tuple, MaterializedField field);
   }
 
+  protected static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(AbstractTupleWriter.class);
```

Comment:
Please use full imports.

Reply:
Done. But note that, when the code was converted from Logback to slf4j, all of the files ended up with the fully qualified class names. Perhaps someone should write a script to change all of these if we wish to use qualified names.

Reply:
Perhaps, maybe someone would like to do it :)